.github/workflows/miniflare-dependabot-versioning-prs.yml

```diff
@@ -25,5 +25,5 @@ jobs:
           git config --global user.email wrangler@cloudflare.com
           git config --global user.name 'Wrangler automated PR updater'
 
-      - name: Generate Miniflare changesets
-        run: node -r esbuild-register tools/dependabot/generate-dependabot-pr-changesets.ts ${{ github.event.number }} miniflare packages/miniflare/package.json dependabot-update
+      - name: Generate Miniflare changeset
```

Comment:
It updates the same PR but the file change that triggers the workflow is the package.json of the package – in this workflow it's miniflare's package.json, in the other it's quick-edit-extension's package.json that triggers the workflow

Reply:
But I agree it would be nicer to have one workflow https://github.com/cloudflare/workers-sdk/pull/6852/commits/107eae2d99bdef036d270664aad5be3cc84a13a9